src/colejorz/pilothouse.py

```diff
@@ -36,7 +39,7 @@ class Pilothouse:  # pylint:disable=too-many-instance-attributes
     STOP = 'stop'
     BACKWARD = 'backward'
 
-    def __init__(self, queue):
+    def __init__(self, queue: "Queue[Dict[str, int]]") -> None:
```

Comment:
Add comment why do we need it in parenthesis

Reply:
And the question if it's still an issue with python 3.7

Reply:
Yes it is, at least when I run tests.